lang/cs/Org.Apache.REEF.Wake/Impl/TimerStage.cs

```diff
@@ -15,14 +15,19 @@
 // specific language governing permissions and limitations
 // under the License.
 
-using System.Timers;
+using System;
+using System.Threading;
 
 namespace Org.Apache.REEF.Wake.Impl
 {
     /// <summary>Stage that triggers an event handler periodically</summary>
     public sealed class TimerStage : IStage
     {
-        // private readonly ScheduledExecutorService executor;
+        enum TimeValue : long
```

Comment:
Why do you need an enum to store a single constant? Using a `long` constant has an added benefit of not having to convert it to `long` explicitly on line 79